azure-devops/.env/terraform.tfvars

```diff
@@ -2,6 +2,14 @@ dev_subscription_name  = "DEV-PAGOPA"
 uat_subscription_name  = "UAT-PAGOPA"
 prod_subscription_name = "PROD-PAGOPA"
 
+project_name        = "pagoPA-projects"
 project_name_prefix = "pagopa"
 
 pipeline_environments = ["DEV", "UAT", "PROD"]
+
```

Comment:
is't necessary to have access at devops prod kv for general info (like tenant, acr, ...)?

Reply:
Yes, to get common service connections, so not dedicated to ecommerce domain, for example: 
- data.terraform_remote_state.app.outputs.service_endpoint_azure_devops_acr_aks_dev_id
- data.terraform_remote_state.app.outputs.service_endpoint_azure_devops_github_ro_id
- data.terraform_remote_state.app.outputs.service_endpoint_azure_dev_id